DEVELOPMENT.md

```diff
@@ -0,0 +1,76 @@
+# DEVELOPMENT
+
+### Environment setup
+
+ - clone the repo
+ - check/install a recent node version >=22
+   `nvm use 22`
+
+- setup python env
+```
+pyenv local 3.12
+poetry env use 3.12
+```
+
+setup yarn 2 ...
+```
+
+
+yarn install
+```
+
+Now `yarn sls info` should print something like ...
+
+```
+chrisbc@tryharder-ubuntu:/GNSDATA/API/kororaa-graphql-api$ sls info
```

Comment:
Is this file copied over from kororaa-graphql-api? If yes, does everything apply to this project?

Reply:
Oh, yes it is, I missed that ref but I did check the file over.